nsis/src/nsis/instructions/OperationInterface.java

```diff
@@ -0,0 +1,24 @@
+package nsis.instructions;
+
+import ghidra.program.model.address.AddressOutOfBoundsException;
+import ghidra.program.model.listing.Instruction;
+import ghidra.program.model.mem.MemoryAccessException;
+import ghidra.program.model.mem.MemoryBlock;
+import ghidra.program.model.symbol.ReferenceManager;
+
+public interface OperationInterface {
```

Comment:
Do you intend to have other classes that will implement `OperationInterface`, but will not subclass `Operation`? If not, then you can simplify the code by removing `OperationInterface` and moving `fixUp` method definition into `Operation`.

Reply:
Good idea, I don't need that interface. I'll move fixUp()